pygfx/show/backends.py

```diff
@@ -0,0 +1,27 @@
+import importlib
+
+
+BACKENDS_QT = ("PySide6", "PyQt6", "PySide2", "PyQt5", "PySide", "PyQt4")
+BACKENDS = ("qt", "wx", "glfw", "jupyter")

```

Comment:
Nitpick: no braces needed :)

Reply:
I'd prefer to be explicit that this is a tuple of possible backends, that the braces are optional is just syntatic sugar